osh/hub/settings.py

```diff
@@ -196,6 +196,9 @@
 # If this setting is enabled, a worker is only used to perform a single task.
 ENABLE_SINGLE_USE_WORKERS = False
 
+# Disable sending messages to Fedora rabbitmq by default
```

Comment:
The `by default` suffix is confusing in this context.  Do you want to highlight that it can be overridden in `settings_local.py`?  That is true for all the options in this file...